.github/workflows/tests.yml

```
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         os: [ 'ubuntu-latest' ] #, macos-latest, windows-latest ]
-        php-version: [ '7.3', '7.4', '8.0', '8.1', '8.2' ]
+        php-version: [ '8.1', '8.2', '8.3' ]
```

Alternatively, in order to save a few cycles, this choice would also be acceptable I guess?

```suggestion
        php-version: [ '7.3', '8.3' ]
```